rc/extra/racer.kak

```diff
@@ -93,3 +93,53 @@ define-command racer-disable-autocomplete -docstring "Disable racer completion"
     remove-hooks window racer-autocomplete
     unalias window complete racer-complete
 }
+
+define-command racer-go-definition -docstring "Jump to where the rust identifier below the cursor is defined" %{
+    %sh{
+        dir=$(mktemp -d "${TMPDIR:-/tmp}"/kak-racer.XXXXXXXX)
+        printf %s\\n "set-option buffer racer_tmp_dir ${dir}"
+        printf %s\\n "evaluate-commands -no-hooks %{ write ${dir}/buf }"
+    }
+    %sh{
+        dir=${kak_opt_racer_tmp_dir}
+        (
```

Comment:
Why create a subshell? If you really need it, consider using `{`/`}` in order not to spawn a new process.

Reply:
Can you expand on this? I'm relatively new to kakoune and I'm not entirely sure I follow your comment.

If you are talking about the two "%sh{" blocks, when I try to put the bit that creates the temporal file in the same block that runs racer, for some reason racer complains the file doesn't exist.

Reply:
The parenthesis line 105 creates a subshell, it's a feature of the shell scripting language. If you don't need the code enclosed by these parenthesis (the closing one is a few lines below) to be run in a separate shell, remove them. Otherwise, consider using curly braces `{`/`}` if possible, because they don't spawn a new process.

Reply:
Thanks, I got it now. Fixed (it works), going to be in on the next commit